config/routes.rb

```diff
@@ -6,7 +6,9 @@
   resources :user_sessions, only: [:new, :create, :destroy]
 
   get 'user_profile' => 'user_profile#edit'
+  put 'user_profile' => 'user_profile#change_locale', as: :change_user_locale
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.